packages/terra-consumer-icon/package.json

```diff
@@ -33,12 +33,10 @@
   "peerDependencies": {
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
```

Comment:
the comma on this line needs to be removed as well @bjankord . Thanks for making this PR to our repo!